lib/logstorage/json_parser.go

```diff
@@ -78,7 +77,7 @@ func (p *JSONParser) parseLogMessage(msg []byte, maxFieldNameLen int) error {
 	msgStr := bytesutil.ToUnsafeString(msg)
 	v, err := p.p.Parse(msgStr)
 	if err != nil {
-		return fmt.Errorf("cannot parse json: %w", err)
```

Comment:
This change is needed to avoid duplicating error messages like:
`cannot parse json: cannot parse JSON ...`